### DISCUSSION TITLE
How does plausible analytics track unique visitors per day?

### DISCUSSION THREAD

**Discussion by gal064** [plausible/analytics#1150] [Anything Else]
That is without using cookies? what's the mechanism to store the fact that this is a repeated user?

**Comment by metmarkosaric** [comment]
you can read more here https://plausible.io/data-policy

**Comment by gal064** [comment]
Thanks! 
Is the random salt the same for all visitors under the domain? if no how would you store the random salt for the day?

**Reply by ukutaht** [reply]
yes the salt is global so it's the same for all visitors
